### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by cairocoder** [aclap-dev/video-downloadhelper#901] [Help and Q&A]
### Your question

Download failed

Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!).

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://nexteg1-my.sharepoint.com/personal/online5_nexteg1_onmicrosoft_com/_layouts/15/stream.aspx?id=%2Fpersonal%2Fonline5%5Fnexteg1%5Fonmicrosoft%5Fcom%2FDocuments%2FRecordings%2FSCM033%2D20240612%5F171033%2DMeeting%20Recording%2Emp4&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&ga=1&referrer=StreamWebApp%2EWeb&referrerScenario=AddressBarCopied%2Eview%2E0601aa7b%2D3674%2D4879%2Da626%2Db7664cf24827",
  "error": {
    "details": "SideDownload error: : [dash @ 0x1458042e0] Error when loading first fragment of playlist\n[in#0 @ 0x1458041d0] Error opening input: Invalid data found when processing input\nError opening input file https://westeurope1-mediap.svc.ms/transform/videomanifest?provider=spo&inputFormat=mp4&cs=fFNQTw&correlationId=2c0e34a1-b060-9000-2ff4-9bb3d95af3db&docid=https%3A%2F%2Fnexteg1-my.sharepoint.com%2Fpersonal%2Fonline5_nexteg1_onmicrosoft_com%2F_api%2Fv2.0%2Fdrives%2Fb%21DmDugYfc_Uy5PoFP15vMGRHKZnC_k_NCqK-xzaf70DAJ5nP24-3XQ5Ctj3cvT0M4%2Fitems%2F01TE3YZIKWNBG7E4CJZNBK362MSY6LWDP4%3Ftempauth%3DeyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiIwMDAwMDAwMy0wMDAwLTBmZjEtY2UwMC0wMDAwMDAwMDAwMDAvbmV4dGVnMS1teS5zaGFyZXBvaW50LmNvbUBiM2FlOTE4NS0xNGVkLTQ5MDQtYmFiNC1iYjg2NTQxZDc2M2YiLCJjYWNoZWtleSI6IjBoLmZ8bWVtYmVyc2hpcHx1cm4lM2FzcG8lM2Fhbm9uIzY5NGE0MzhmMTViZjk4Y2QxYWNlYWYyZmIxYzU1MGU1YTUwZWFjM2RkNzI3MWJiYjA2YjJlNWFlNDRkMGJiNzciLCJjaWQiOiJvVFFPTEdDd0FKQXY5SnV6MlZyejJ3PT0iLCJlbmRwb2ludHVybCI6ImlxQ09FMlNrN3hxcVhuRXlBelpaWmNDYkpwS24rMURwMWwwZDhtaVpJMUE9IiwiZW5kcG9pbnR1cmxMZW5ndGgiOiIxOTkiLCJleHAiOiIxNzE4NzkyMTkzIiwiaXBhZGRyIjoiMjAzLjE1OS44MS4zNCIsImlzbG9vcGJhY2siOiJUcnVlIiwiaXNzIjoiMDAwMDAwMDMtMDAwMC0wZmYxLWNlMDAtMDAwMDAwMDAwMDAwIiwiaXN1c2VyIjoidHJ1ZSIsIm5hbWVpZCI6IjAjLmZ8bWVtYmVyc2hpcHx1cm4lM2FzcG8lM2Fhbm9uIzY5NGE0MzhmMTViZjk4Y2QxYWNlYWYyZmIxYzU1MGU1YTUwZWFjM2RkNzI3MWJiYjA2YjJlNWFlNDRkMGJiNzciLCJuYmYiOiIxNzE4NzcwNTkzIiwibmlpIjoibWljcm9zb2Z0LnNoYXJlcG9pbnQiLCJzaGFyaW5naWQiOiJqelVqY0lPOGhrYUxuclNUZUh3SkxnIiwic2l0ZWlkIjoiT0RGbFpUWXdNR1V0WkdNNE55MDBZMlprTFdJNU0yVXRPREUwWm1RM09XSmpZekU1IiwidHQiOiIwIiwidmVyIjoiaGFzaGVkcHJvb2Z0b2tlbiJ9.uHd0OfGGcPG_sXutQngMHiW6swScoF3nUdVLdB1QgCs%26version%3DPublished&psi=0601aa7b-3674-4879-a626-b7664cf24827&enhanceAudio=true&cTag=%22c%3A%7BF24D6856-4970-42CB-ADFB-4C963CBB0DFC%7D%2C2%22&action=Access&part=index&format=dash&altManifestMetadata=eyJBdWRpb0NvZGVjSW5mbyI6ImFhYyIsIkF1ZGlvRnJhbWVTaXplIjoxMDI0LCJBdWRpb1NhbXBsaW5nUmF0ZSI6MTYwMDAsIkF1ZGlvU3RhcnRUaW1lSW4xMDBOYW5vIjowLCJBdWRpb1N0cmVhbUR1cmF0aW9uMTAwTmFubyI6Nzc4MTMxMjAwMDAsIkJpdHJhdGUiOjY5Njc3MSwiQ1RhZyI6bnVsbCwiQ29udGFpbmVyU3RhcnRUaW1lSW4xMDBOYW5vIjowLCJDb250YWluc0F1ZGlvIjp0cnVlLCJDb250YWluc1ZpZGVvIjp0cnVlLCJDb250ZW50QXdhcmVCaXRyYXRlIjpudWxsLCJDb250ZW50Q2F0ZWdvcnkiOiJ0ZWFtc19tZWV0aW5nIiwiQ3JlYXRpb25Vbml4VGltZXN0YW1wU2Vjb25kcyI6MTcxODIxMjM4NCwiQ3J5cHRvZ3JhcGhpY0RhdGFDcmVhdGVkRGF0ZVVuaXgiOjE3MTgyMTIzODQsIkNyeXB0b2dyYXBoaWNEYXRhSVYiOiIweDg3OEY3NTAwMjUyRDMwNDQ0QTU2NUE0OUEyRkE5QzIzIiwiQ3J5cHRvZ3JhcGhpY0RhdGFLaWQiOiI3OTQ5NTFhZGJiOS45NDc4MDAuNjg5NzQiLCJEdXJhdGlvbjEwME5hbm8iOjc3ODEzNzUwMDAwLCJGcmFtZVNpemUiOnsiaGVpZ2h0IjoxMDgwLCJ3aWR0aCI6MTkyMH0sIkdvcFNpemUiOjYwMDAwMDAwLCJIMjY0TGV2ZWwiOjQwLCJIYXNCRnJhbWVzIjpmYWxzZSwiSGVhZGVyT2Zmc2V0SW5CeXRlcyI6MjQsIkhlYWRlclNpemUiOjIyMDQ5ODgsIk1ldGFkYXRhU291cmNlSW5mb3JtYXRpb24iOnsiQXVkaW9TdHJlYW1EdXJhdGlvblNvdXJjZSI6MSwiQXVkaW9TdHJlYW1SZWFkQnVmZmVyU2l6ZUluQnl0ZXMiOjE2Mzg0LCJDb250YWluZXJEdXJhdGlvblNvdXJjZSI6MSwiVmlkZW9CaXRyYXRlU291cmNlIjoxLCJWaWRlb1N0cmVhbUR1cmF0aW9uU291cmNlIjoxLCJWaWRlb1N0cmVhbVJlYWRCdWZmZXJTaXplSW5CeXRlcyI6MTYzODR9LCJQYWNrZWRGcmFtZVJhdGUiOnsiSGlnaCI6MTYsIkxvdyI6MX0sIlByZW1pdW1WaWRlb1ByZXRyYW5zY29kZURhdGEiOm51bGwsIlJlYWxCYXNlRnJhbWVSYXRlIjp7IkhpZ2giOjE2LCJMb3ciOjF9LCJVcGRhdGVhYmxlQXVkaW9UcmFja3MiOltdLCJWZXJzaW9uIjp7Il9CdWlsZCI6LTEsIl9NYWpvciI6NiwiX01pbm9yIjowLCJfUmV2aXNpb24iOi0xfSwiVmlkZW9Db2RlY0luZm8iOiJoMjY0IiwiVmlkZW9QcmV0cmFuc2NvZGVEYXRhIjp7IkF1ZGlvUXVhbGl0eUlkcyI6ImF1ZGNvcHkiLCJTZWdtZW50Q291bnQiOjIsIlZpZGVvUXVhbGl0eUlkcyI6InZjb3B5In0sIlZpZGVvU3RhcnRUaW1lSW4xMDBOYW5vIjowLCJWaWRlb1N0cmVhbUR1cmF0aW9uMTAwTmFubyI6Nzc4MTM3NTAwMDB9&altTranscode=1&useScf=true&pretranscode=0&transcodeahead=0&enableCdn=1&hybridPlayback=false&oif=4kp&pn=hostApp-OnePlayer-Web.\nError opening input files: Invalid data found when processing input\n",
    "downloadable_id": "downloadable_3094392472993444",
    "error": "coapp_failure",
    "id": "coapp_failure_4d32a51d-d03e-4801-a69b-99ad0ebbe2bf"
  }
}

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"2a77a46806a14b85"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 11,
    "successfull_dl": 15
}

**Comment by Mattmjs** [comment]
Unfortunately this sharepoint video is protected by DRM which means downloading is not possible. I used your input file in
another tool and this was the result:

**ERROR:** [generic] videomanifest?provider=spo&inputFormat=mp4&cs=fFNQTw&correlationId=2c0e34a1-b060-9000-2ff4-9bb3d95af3db&docid=https://nexteg1-my.sharepoint.com/personal/online5_nexteg1_onmicrosoft_com/_api/v2.0/drives/b!DmDugYfc_Uy5PoFP15vMGRHKZnC_k_NCqK-xzaf70DAJ5nP24-3XQ5Ctj3cvT0M4/items/01TE3YZIKWNBG7E4CJZNBK362MSY6LWDP4?tempauth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiIwMDAwMDAwMy0wMDAwLTBmZjEtY2UwMC0wMDAwMDAwMDAwMDAvbmV4dGVnMS1teS5zaGFyZXBvaW50LmNvbUBiM2FlOTE4NS0xNGVkLTQ5MDQtYmFiNC1iYjg2NTQxZDc2M2YiLCJjYWNoZWtleSI6IjBoLmZ8bWVtYmVyc2hpcHx1cm4lM2FzcG8lM2Fhbm9uIzY5NGE0MzhmMTViZjk4Y2QxYWNlYWYyZmIxYzU1MGU1YTUwZWFjM2RkNzI3MWJiYjA2YjJlNWFlNDRkMGJiNzciLCJjaWQiOiJvVFFPTEdDd0FKQXY5SnV6MlZyejJ3PT0iLCJlbmRwb2ludHVybCI6ImlxQ09FMlNrN3hxcVhuRXlBelpaWmNDYkpwS24rMURwMWwwZDhtaVpJMUE9IiwiZW5kcG9pbnR1cmxMZW5ndGgiOiIxOTkiLCJleHAiOiIxNzE4NzkyMTkzIiwiaXBhZGRyIjoiMjAzLjE1OS44MS4zNCIsImlzbG9vcGJhY2siOiJUcnVlIiwiaXNzIjoiMDAwMDAwMDMtMDAwMC0wZmYxLWNlMDAtMDAwMDAwMDAwMDAwIiwiaXN1c2VyIjoidHJ1ZSIsIm5hbWVpZCI6IjAjLmZ8bWVtYmVyc2hpcHx1cm4lM2FzcG8lM2Fhbm9uIzY5NGE0MzhmMTViZjk4Y2QxYWNlYWYyZmIxYzU1MGU1YTUwZWFjM2RkNzI3MWJiYjA2YjJlNWFlNDRkMGJiNzciLCJuYmYiOiIxNzE4NzcwNTkzIiwibmlpIjoibWljcm9zb2Z0LnNoYXJlcG9pbnQiLCJzaGFyaW5naWQiOiJqelVqY0lPOGhrYUxuclNUZUh3SkxnIiwic2l0ZWlkIjoiT0RGbFpUWXdNR1V0WkdNNE55MDBZMlprTFdJNU0yVXRPREUwWm1RM09XSmpZekU1IiwidHQiOiIwIiwidmVyIjoiaGFzaGVkcHJvb2Z0b2tlbiJ9: **This video is DRM protected**
